pkg/graveler/committed/merge_test.go

```diff
@@ -66,18 +67,35 @@ type testRunResult struct {
 	expectedErr     error
 }
 
+type testMetaRange struct {
```

Comment:
can be []testRange this way we don't need special construction for all the test code.
```go
type testMetaRange []testRange
```